package.json

```diff
@@ -9,6 +9,11 @@
     "type": "git",
     "url": "https://github.com/RentPath/jquery-autotagging.git"
   },
+  "peerDependencies": {
```

Comment:
`peerDependencies` is deprecated in npm 3. Should these be strict dependencies?

Further reading: https://docs.npmjs.com/files/package.json#peerdependencies